### ISSUE TITLE
Replacement of ic-deps?

### ISSUE THREAD

**Issue by benwhalley** [bigskysoftware/htmx#21]
I was a real fan of ic-deps as a way of allowing pages to, effectively, subscribe to changes which occur on the server-side after a form post is processed.

After a bit of back and forth on the gitter channel @chg20 suggested: `hx-trigger="subscribe/foo/bar"` as a possible replacement, or `hx-trigger="subscribe:POST/foo/bar"` if you only wanted to update for particular verbs. I'd be an enthusiastic supporter of this as it makes what was nice about ic-deps even more explicit.

**Comment by chg20** [comment]
@benwhalley if you are interested I think this could be implemented as an extension in the new extension infrastructure I just added.  

Something like this:

```html
<body hx-ext="path-deps">
  <div hx-trigger="dependencies" path-deps="/foo/bar">...</div>
  <button hx-post="/foo/bar">Post It!</button>
</body>
```
Are you interested in taking a crack at that?

**Comment by chg20** [comment]
ok, added a `path-deps` extetension with https://github.com/bigskysoftware/htmx/commit/23a9a33486075160d8faa3b9dccf78fdccb61a93 

will be available in `0.0.4`
